### PR TITLE
feature 5: A unit test for cpu monitoring script

### PR DESCRIPTION
**Pull request by Elwizzy12**
feature 5: A unit test for cpu monitoring script

### CODE REVIEW

File: services/network_service.py
Comment (stephane-segning) [inline services/network_service.py:1]: Why is this change here? Did you implemented the network?

File: services/tests/process_tracking.py
Comment (stephane-segning) [inline services/tests/process_tracking.py:1]: Do this change have something to do with this ticket?

File: services/tests/test_process_tracking.py
Comment (stephane-segning) [inline services/tests/test_process_tracking.py:1]: Do this change have something to do with this ticket?

File: static/js/polling.js
Comment (stephane-segning) [inline static/js/polling.js:1]: Do this change have something to do with this ticket?

File: templates/index.html
Comment (stephane-segning) [inline templates/index.html:1]: Do this change have something to do with this ticket?

File: doc/README.md
Comment (stephane-segning) [inline doc/README.md:1]: Do this change have something to do with this ticket?

File: app.py
Comment (stephane-segning) [inline app.py:1]: Do this change have something to do with this ticket?

File: ticket#42
Comment (stephane-segning) [inline ticket#42:1]: This is not a change for this ticket (the name speaks for itself)

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work, but wrong work @Elwizzy12! Try and review these points:

**Comment by Elwizzy12** [comment]
Sir, I just realized I made an error.
I have just correct it and resolved all the conflicts

I'm waiting for your permission now to merge.
Please do verify!
Thank you.
